packages/design-system/src/utils/form-event-bus.ts

```diff
@@ -0,0 +1,12 @@
+import { createEventBus } from './event-bus';
+
+export interface FormEventBusEvents {
+	submit: never;
+}
+
+export type FormEventBus = ReturnType<typeof createFormEventBus>;
+
+/**
+ * Creates a new event bus to be used with the `FormInputs` component.
+ */
+export const createFormEventBus = createEventBus<FormEventBusEvents>;
```

Comment:
considering that we have so many other instances of `createEventBus`, do we plan to create a typed alias for each of them? if not, perhaps we should remove this, and use `createEventBus<FormEventBusEvents>()` instead of `createFormEventBus()` ?

Reply:
I think it makes sense to have an aliased create function for cases where it's used for a common component. For example for the `FormInputs` and for modals. I could move this to the `FormInputs` component file so it's closer to usage.

Reply:
I still think that once we migrate all other event buses to be typed, creating aliases for each would seem too verbose. but we can talk about that after we finish porting over the rest of the code.